i18n/locales/source/course_content/2019/csd5-2019.json

```diff
@@ -45,27 +45,27 @@
   },
   "https://studio.code.org/s/csd5-2019/stage/5/puzzle/2": {
     "long_instructions": "# 5-Bit Number\n\nWhen you only had four bits, the highest number that you could make was fifteen.  Now you'll have a chance to work with more bits to make higher numbers.\n\nIn this activity, the dots are replaced with numbers, but you still add up everything that's showing to see what number the binary code stands for.  The left-most bit doesn't have a number yet, so you'll have to give it one.\n\n# Do This\n\n-   Click on the bits to turn them on and off, and see the different numbers you can make.\n-   With your partner, discuss the pattern in the numbers.  Decide what number you think should go in the left most box.\n-   Click \"Edit\" to change the value of the left most bit.\n-   Once you've decided on the value, try to make the following numbers: 16, 27, 31",
-    "teacher_markdown": "Answers for this level can be found in the exemplar linked on [Level 1][0]."
+    "teacher_markdown": "[][0]\n\nAnswers for this level can be found in the exemplar linked on [Level 1][1].\n\n[/][0]"
```

Comment:
@Hamms will this break existing translations?

Reply:
Actually, I can't find any translations for this strings so probably fine

Reply:
oh, good catch! It probably would have, but disaster avoided!